### PR TITLE
[action] add claim reward account

### PR DESCRIPTION
**Pull request by saitofun**
# Description

1. add `address` field to `action.ClaimFromRewardingFund` to assign the address claim reward to (6efdfb2)
2. fix unit tests (426ba7d)
3. add `claim to` arg for `ioctl action claim` (fab352e)
4. check address when handle claim reward, if `address` is empty, default use action's caller address.(2aea4c5)

Fixes #(issue)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
- [x] make test

**Test Configuration**:
- Firmware version:
- Hardware:
- Toolchain:
- SDK:

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by CoderZhi — APPROVED** [review]

File: action/claimreward.s
[the remark this answers is not in the document]
Comment (saitofun) [reply action/claimreward.s:1]: for exporting private var or function from `action` package
https://github.com/iotexproject/iotex-core/blob/2aea4c56f74766a0fa8d0f0ec133774cb6d41110/action/claimreward_test.go#L19

File: ioctl/cmd/action/actionclaim.go [hunk 1]
```diff
@@ -21,16 +21,16 @@ var (
 		config.Chinese: "从奖励基金中获取奖励",
 	}
 	_claimCmdUses = map[config.Language]string{
-		config.English: "claim AMOUNT_IOTX [DATA] [-s SIGNER] [-n NONCE] [-l GAS_LIMIT] [-p GAS_PRICE] [-P PASSWORD] [-y]",
-		config.Chinese: "claim IOTX数量 [数据] [-s 签署人] [-n NONCE] [-l GAS限制] [-p GAS价格] [-P 密码] [-y]",
+		config.English: "claim AMOUNT_IOTX [ACCOUNT_REWARD_TO] [DATA] [-s SIGNER] [-n NONCE] [-l GAS_LIMIT] [-p GAS_PRICE] [-P PASSWORD] [-y]",
```
[the remark this answers is not in the document]
Comment (saitofun) [reply ioctl/cmd/action/actionclaim.go:24]: the `payload` arg is not required?

[the remark this answers is not in the document]
Comment (saitofun) [reply ioctl/cmd/action/actionclaim.go:24]: yes, this two args should assigned by flag name

File: ioctl/cmd/action/actionclaim.go [hunk 2]
```diff
@@ -26,38 +28,58 @@ var (
 	}
 )
 
+// flags
+var (
+	claimAmount  = flag.NewStringVarP("amount", "", "0", config.TranslateInLang(_flagClaimAmount, config.UILanguage))
```
Comment (envestcc) [inline ioctl/cmd/action/actionclaim.go:33]: Keep consistent with cmd usage

**Review by envestcc — APPROVED** [review]

**Pull request merged**